webcompat/static/css/src/issue-wizard.css

```diff
@@ -498,6 +498,10 @@
   padding: 5px 32px 7px 8px;
 }
 
+.issue-form .is-error .text-field {
+  border-color: var(--wizard-step-error);
```

Comment:
So we also need to remove the old rule: 
https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/css/src/issue-wizard.css#L1040
Since here you have it as a default style, it will apply both to mobile and desktop, so having it within `@media (min-width: 665px)` is unnecessary.

Given that this branch doesn't have the old rule, you probably need to rebase this branch with master.